docs/design/thee_datahall.md

```diff
@@ -0,0 +1,197 @@
+# Three datahall and three datacenter redundancy mode
+
+## Metadata
+
+* Authors: @johscheuer
+* Created: 2020-07-05
+* Updated: 2020-07-05
+
+## Background
+
+Many Cloud Provder offer regional Kubernetes clusters that are span across multiple avalability zones (AZ).
+An avalability zone is an isolated zone in a region and offers users the ability ro create systems that run atop of these failure domains.
+For on-premise clusters this could be a rack, a datahall or a data center.
+The current implementation of the operator only supports either single FoundationDB clusters with the redundancy modes `single`, `dobule`, `triple` or HA clusters span across multiple Kubernetes clusters.
+FoundationDB offers the redundnacy mode [three_data_hall](https://apple.github.io/foundationdb/configuration.html#single-datacenter-modes) or [three_datacenter](https://apple.github.io/foundationdb/configuration.html#datacenter-aware-mode) that fits well in these enviornments without the overhead of the HA solution.
+The current requirement for `three_data_hall` and `three_datacenter` is to have 3 AZs and at least two different failure zones per `data_hall`/`datacenter`.
+
+## General Design Goals
+
+The goal of this design is to support the redundancy modes `three_data_hall` and `three_datacenter` in different deployment models:
```

Comment:
I'm happy to remove it I just thought it makes sense to support it (I'm also not sure about the state) since the requirements are similar to `three_data_hall`. The benefit would be to have 6 replicas for the storage system instead of 3.